src/components/messaging-setup-button.tsx

```diff
@@ -77,24 +76,20 @@ export const MessagingSetupButton = () => {
       onClick={async () => {
         setIsSettingUpMessaging(true);
         try {
-          const tx = await desoAPI.accessGroup.CreateAccessGroup(
-            {
-              AccessGroupOwnerPublicKeyBase58Check:
-                appUser.PublicKeyBase58Check,
-              AccessGroupPublicKeyBase58Check:
-                appUser.messagingPublicKeyBase58Check,
-              AccessGroupKeyName: DEFAULT_KEY_MESSAGING_GROUP_NAME,
-              MinFeeRateNanosPerKB: 1000,
-            },
-            {
-              broadcast: false,
-            }
-          );
-
-          await identity.signAndSubmit(tx);
+          // TODO: can we do this automatically
```

Comment:
@lazynina to check

Reply:
ahh these are old comments. I just deleted them.